### DISCUSSION TITLE
Ten Tips to Make Conference Talks Suck Less - Gunnar Morling

### DISCUSSION THREAD

**Discussion by giscus** [gunnarmorling/discussions.morling.dev#297] [Announcements]
# Ten Tips to Make Conference Talks Suck Less - Gunnar Morling

Every so often, I come across some conference talk which is highly interesting in terms of its actual contents, but which unfortunately is presented in a less than ideal way. I’m thinking of basic mistakes here, such as the presenter primarily looking at their slides rather than at the audience. I’m always feeling a bit sorry when this happens, as I firmly believe that everyone can do good and even great talks, just by being aware of — and thus avoiding — a few common mistakes, and sticking to some simple principles.

https://www.morling.dev/blog/ten-tips-make-conference-talks-suck-less/

**Comment by orevial** [comment]
Awesome, I recently gave an introduction on "how to prepare and give a talk" at my company with pretty much all the points you mentioned, I can only approve !
I have a few other points :
- Make sure images you use in your talk are not copyrighted
- Avoid gifs. While they can add fun and sometimes carry some specific emotion, they can also make your audience totally lose what you are saying because they will focus on this hyptonic and way-too-repetitive gifs
- While rehearsing, note your chapter times. It will help you better size your chapters for the actual talk, reducing or adding content as you see that you have way too much content (or not enough).
- While rehearsing, note the difficult parts on which you tend to stumble. You can later insist on those specific parts on following rehearsals
- Rehearse in front of people before the actual talk if you can. I usually give my public talks internally in my company first so I can test it in front of a real audience (with less stress than the actual public conference though), and I can get a first gentle feedback from my colleagues
- Do small breaks as you talk. For example, take 10 seconds to drink a sip of water. That way you can catch your breath if you were going too fast, and it will also make the audience retain what you just said.
- If you get questions after your talk : don't bullshit if you don't know. People do not expect you to know each detail of your subject at 100%, so if you don't know, just say it.

**Reply by JohannesDienst** [reply]
Last point is so important! It usually taints the whole talk if some tries to bullshit out of a question.

**Reply by bertjan** [reply]
Thanks Gunnar for sharing, and thanks Olivier for the additions!
I'll happily use those for a speakering program I run in the NLJUG - and credit you of course ;-)

**Reply by gunnarmorling** [reply]
Great additions, @orevial, thanks a lot!

@bertjan, happy to hear you find this resource useful. Good luck for NLJUG, heard lots of good things about.

**Reply by orevial** [reply]
You're welcome, I might write my own article on this, you inspired me 🙂 Thank you !
